### PR TITLE
#16-log-version

### LINKED ISSUE #16: Add driver version to log output and client info
For debugging issues it is helpful to know the exact driver version. That's why we should write the driver version to the log and also use it in the client info when connecting to the database.

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #16

### CODE REVIEW

File: src/metabase/driver/exasol.clj [hunk 1]
```diff
@@ -18,6 +19,28 @@
...
+(defn get-jdbc-driver-version []
+  (com.exasol.jdbc.EXADriver/getVersionInfo))
+
+(defn get-driver-version
+  ([]
+   (get-driver-version "META-INF/maven/metabase/exasol-driver/pom.properties"))
+  ([resource]
+   (when-let [url ^java.net.URL (io/resource resource)]
+     (with-open [stream (.openStream url)]
+       (let [p (java.util.Properties.)]
```
Comment (jakobbraun) [inline src/metabase/driver/exasol.clj:32]: can you rename p?

File: src/metabase/driver/exasol.clj [hunk 2]
```diff
@@ -18,6 +19,28 @@
...
+         (try
+           (.load p stream)
+           (.getProperty p "version")
+           (catch Exception _)))))))
+
+(defn- log-driver-version []
+  (log/info (u/format-color 'green (format "Loading Exasol Metabase driver %s, Exasol JDBC driver: %s"
```
Comment (jakobbraun) [inline src/metabase/driver/exasol.clj:39]: is it a good practice to use colors in logs?

Comment (kaklakariada) [reply src/metabase/driver/exasol.clj:39]: The Metabase log is very colorful:
image

File: src/metabase/driver/exasol.clj (hunk 1 above)
Comment (kaklakariada) [reply src/metabase/driver/exasol.clj]: ✅

**Review by jakobbraun — APPROVED** [review]

**Pull request merged**